### DISCUSSION TITLE
many to many?

### DISCUSSION THREAD

**Discussion by MartinJohannessen** [dineug/erd-editor#351] [Q&A]
This is a great tool for learning data modelling, thanks!

BUT why is there no many to many relationship?

**Comment by dineug** [comment]
This tool is a physical database modeling tool.
N:M relationships are modeled as follows.
<img width="293" alt="n_m" src="https://github.com/dineug/erd-editor/assets/45829489/d1ff6bd8-a59d-4077-8eaa-c51b1d419121">


The relationship names like "zero one" refer to the states of the PK and the FK it's pointing to, not the PK state itself.

Exceptionally, the tool accommodates the [Crow's foot notation](https://en.wikipedia.org/wiki/Entity%E2%80%93relationship_model#Crow's_foot_notation) for relationships.

If the relationship notation were purely physical, only the following notation would exist.
![image](https://github.com/dineug/erd-editor/assets/45829489/bc144e59-969e-4723-95b1-6ce9e187858f)

**Comment by MartinJohannessen** [comment]
Thanks so much! This tool forced me to figure out good practices.
